src/variant.h

```diff
@@ -90,6 +90,7 @@ struct Variant {
   bool mustCapture = false;
   bool mustDrop = false;
   PieceType mustDropType = ALL_PIECES;
+  bool isPriorityDrop[PIECE_TYPE_NB] = {};
```

Comment:
It would be more consistent and storage efficient to use a `PieceSet` here, similar as for properties. This would also simplify the parsing, since it can reuse existing logic. It could perhaps also already be made color-specific, similar as `promotionPieceTypes` or the like.